openedx_webhooks/lib/edx_repo_tools_data/test/models/conftest.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import datetime
+
+import pytest
+
+from openedx_webhooks.lib.edx_repo_tools_data.models import People, Person
+
+
+@pytest.fixture
+def active_data():
+    data = {'active-person': {
+        'agreement': 'individual',
+    }}
+    return data
+
+
+@pytest.fixture
+def expired_data():
+    data = {'expired-person': {
+        'agreement': 'institution',
+        'expires_on': datetime.date(2012, 10, 1),
+        'institution': 'edX',
+    }}
+    return data
+
+
+@pytest.fixture
+def active_person(active_data):
+    for k, v in active_data.items():
+        person = Person(k, v)
```

Comment:
This looks like a mistake: it will loop over "all" of active_data, but only make one Person.  I understand that active_data only has one item, but looping is odd here. We need some acknowledgement here of what is going on.  You don't want popitem because that will mutate the dict. Maybe `k, v = next(active_data.items())`, with a comment about there being only one item in active_data? Or maybe an assert on the length of active_data?